storybook/stories/staking/StakePools.stories.js

```diff
@@ -6,7 +6,10 @@ import { action } from '@storybook/addon-actions';
 import StakePools from '../../../source/renderer/app/components/staking/stake-pools/StakePools';
 import STAKE_POOLS from '../../../source/renderer/app/config/stakingStakePools.dummy.json';
 
-export const StakePoolsStory = (props: { currentTheme: string }) => (
+export const StakePoolsStory = (props: {
```

Comment:
Just a minor thing.

Maybe extract props at the top, like on other stores.

type Props = {
  currentTheme: string,
  isLoading: boolean,
};

Then just use
export const StakePoolsStory = (props: Props) {

Reply:
@DeeJayElly may I kindly ask you to implement this change please?
@tomothespian has other work at the moment.

Reply:
@nikolaglumac @tomothespian done